components-sdk/src/locales/es.json

```diff
@@ -0,0 +1,63 @@
+{
+  "components.content": "Texto",
+  "components.container": "Contenedor",
+  "components.image": "Imagen",
+  "components.file": "Archivo",
+  "components.separator": "Separador",
+  "components.button": "Botón",
+  "components.button-link": "Botón de enlace",
+  "components.select-menu": "Menú de selección",
+  "components.thumbnail": "Miniatura",
+  "components.text-input": "Entrada de texto",
+  "button.add-component": "Agregar componente",
+  "button.add-inline": "Añadir componente debajo",
+  "button.add-content": "Añadir contenido",
+  "button.add-button": "Añadir botón",
+  "button.new-option": "Nueva opción",
+  "error": "Error:",
+  "actions.add-action": "Añadir acción",
+  "file.upload-file": "Click para subir un archivo",
+  "separator.margin": "Haz clic para cambiar el margen",
+  "thumbnail.set-image-url": "Usar imagen de internet",
+  "thumbnail.image-url": "URL de la imagen, https://...",
+  "thumbnail.upload-image": "Subir imagen",
+  "thumbnail.add-description": "Añadir texto alternativo",
+  "thumbnail.change-description": "Cambiar texto alternativo",
+  "thumbnail.clear-description": "Borrar texto alternativo",
+  "thumbnail.set-spoiler": "Establecer spoiler",
+  "thumbnail.remove-spoiler": "Quitar spoiler",
```

Comment:
```suggestion
  "thumbnail.set-spoiler": "Marcar como spoiler",
  "thumbnail.remove-spoiler": "Quitar como spoiler",
```

Reply:
Spoiler is an accepted Spanish word as part of the RAE, and it can be used as a subject, so grammatically it makes sense. I have seen in it stated like this in other places in the web

It also happens to be what the Discord client uses.
![Screenshot_20250815_234616_Discord.jpg](https://github.com/user-attachments/assets/8c1c682c-4e4d-4cc6-8e37-d81bf6a40999)